Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,18 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
+{
+    for (unsigned i = 0; i < str.length(); i++) {
+        auto currentChar = str[i];
+        if (currentChar == 59 || currentChar == 127)
```

Comment:
I think we tend to prefer to not have "magic" values like these, and instead have named constants.

Reply:
That's a good point--I changed it to hex as recommended below--and since these are simply unicode character codes, I added a comment explaining where they come from.